src/components/Footer.js

```diff
@@ -142,30 +145,40 @@ function Footer({ classes }) {
       <div className={classes.top}>
         <Grid item xs={12} md={8}>
           <Typography className={classes.footerText}>
-            <span className={classes.highlight}>Sea Sensors</span> is an
-            acoustic data project that collects underwater blast data to help
-            provide quantitative spatial information on the incidence of the
-            problem over time, that will enable targeted enforcement.
+            <a href="https://alpha.seasensors.africa/" className={classes.link}>
+              <span className={classes.highlight}>Sea Sensors</span>{' '}
+            </a>
+            is an acoustic data project that collects underwater blast data to
+            help provide quantitative spatial information on the incidence of
+            the problem over time, that will enable targeted enforcement.
           </Typography>
           <Typography className={classes.footerText}>
             The project is funded through{' '}
-            <span className={classes.highlight}>Code for Africa</span>’s{' '}
-            <span className={classes.highlight}>innovateAFRICA</span> programme.
+            <span className={classes.highlight}>Code for Africa</span>
+            ’s <span className={classes.highlight}>innovateAFRICA</span>{' '}
+            programme.
           </Typography>
           <Typography className={classes.footerText}>
-            <span className={classes.highlight}>Code for Africa</span>{' '}
+            <a href="https://codeforafrica.org" className={classes.link}>
```

Comment:
All **external** links should open in new tabs. Use [A.js](https://github.com/CodeForAfrica/sensors.AFRICA-SeaSensors/blob/master/src/components/A.js) component.